sunpy/net/vso/vso.py

```diff
@@ -40,8 +41,10 @@
 
 TIME_FORMAT = config.get("general", "time_format")
 
-DEFAULT_URL = 'http://docs.virtualsolar.org/WSDL/VSOi_rpc_literal.wsdl'
-DEFAULT_PORT = 'nsoVSOi'
+DEFAULT_URL_PORT = {
```

Comment:
this would be better as a list of dictionaries, rather than a dict of dicts like this.